roles/confluent.kafka_connect/tasks/confluent_hub.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Creating Confluent Hub Install Dir
+  file:
+    path: "{{kafka_connect_confluent_hub_client.install_dir}}"
+    state: directory
+    group: "{{kafka_connect_group}}"
+    owner: "{{kafka_connect_user}}"
+    mode: '764'
+  
+- name: Installing/Unarchiving Confluent Hub
+  unarchive:
+    src: "{{kafka_connect_confluent_hub_client.file}}"
+    dest: "{{kafka_connect_confluent_hub_client.install_dir}}"
+    remote_src: "{{kafka_connect_confluent_hub_client.is_remote}}"
+    #group: "{{kafka_connect_group}}"
+    #owner: "{{kafka_connect_user}}"
+    creates: "{{kafka_connect_confluent_hub_client.install_dir}}/bin/confluent-hub"
+
+- name: Symlinking Confluent Hub CLI
+  file:
+    src: "{{kafka_connect_confluent_hub_client.install_dir}}/bin/confluent-hub"
+    dest: "/usr/bin/confluent-hub"
+    state: link
+    #group: "{{kafka_connect_group}}"
+    #owner: "{{kafka_connect_user}}"
+    
+- name: "Installing Kafka Connect Connector(s) from Confluent Hub"
+  shell: "{{kafka_connect_confluent_hub_client.install_dir}}/bin/confluent-hub install --no-prompt --component-dir {{item.dest}} {{item.connector}}"
```

Comment:
Regarding the other comment: what im saying is right here you do have `{{item.dest}}` my point is couldn't we just choose a directory for them? as a customer the user experience should be- "yes i want connectors installed form confluent hub, and these are the ones"

i can see the reason for customizing the install directory, so make that a variable with a default